Cargo.toml

```diff
@@ -20,16 +21,16 @@ path = "src/lib.rs"
 
 [dependencies]
 bytes = "0.4"
```

Comment:
Should be 0.5

Reply:
We cannot do now since prost `0.5`'s `decode` has `IntoBuf` as trait bound. We should wait for https://github.com/danburkert/prost/commit/ece64fa021238b2a381ba0a3a4b3225220a4de46 to be included in a new release. Or we can depend on prost with git rev if you want.

Reply:
I see. Ok, 0.4 is fine